app/assets/javascripts/student_profile/TransitionNotes.js

```diff
@@ -54,52 +54,17 @@ class TransitionNotes extends React.Component {
     this.state = {
       noteText: (regularNote ? regularNote.text : notePrompts),
       restrictedNoteText: (restrictedNote ? restrictedNote.text : restrictedNotePrompts),
-      regularNoteAutosaveStatus: null,
-      restrictedNoteAutosaveStatus: null,
     };
 
-    const throttleOptions = {leading: true, trailing: true};
+    const throttleOptions = {leading: false, trailing: true};
 
-    this.autosaveRegularNote = _.throttle(this.autosaveRegularNote.bind(this), 3000, throttleOptions);
-    this.autosaveRestrictedNote = _.throttle(this.autosaveRestrictedNote.bind(this), 3000, throttleOptions);
+    this.autosaveRegularNote = _.throttle(this.autosaveRegularNote.bind(this), 1000, throttleOptions);
+    this.autosaveRestrictedNote = _.throttle(this.autosaveRestrictedNote.bind(this), 1000, throttleOptions);
 
     this.onChangeRegularNote = this.onChangeRegularNote.bind(this);
     this.onChangeRestrictedNote = this.onChangeRestrictedNote.bind(this);
   }
 
-  // Note on componentWillReceiveProps:
```

Comment:
👍 simplifying!

Reply:
👍 — yup, I tried to get way too clever hear and it definitely made the user experience worse!